cr-usecases/src/main/java/com/thefuntasty/mvvm/crusecases/CoroutineScopeOwner.kt

```diff
@@ -79,6 +80,42 @@ interface CoroutineScopeOwner {
         }
     }
 
+    /**
+     * Synchronously executes use case and saves it's Deferred. By default all previous
+     * pending executions are canceled, this can be changed by the [cancelPrevious].
+     * This version is used for use cases without initial arguments.
+     *
+     * @return [Result] that encapsulates either a successful result with [Success] or a failed result with [Error]
+     */
+    suspend fun <T : Any?> UseCase<Unit, T>.execute(cancelPrevious: Boolean = true): Result<T> {
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/1e22aaaf9def5bbeda391f2cd9735be2fc018fbd/cr-usecases/src/main/java/com/thefuntasty/mvvm/crusecases/CoroutineScopeOwner.kt#L90"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>